sphinxext/rediraffe.py

```diff
@@ -319,57 +320,34 @@ def abs_path_in_src_dir_w_src_suffix(filename: str) -> Union[Path, None]:
-                    logger.warning(
-                        f"{red('(broken)')} Automatic redirects is only available with a redirects file."
-                    )
-                elif (
-                    self.app.config.rediraffe_auto_redirect
-                    and "rediraffewritediff" in self.name
-                ):
-                    if path_rename_from not in absolute_redirects:
-                        if perc >= self.app.config.rediraffe_auto_redirect_perc:
-                            from_fancy_path = str(renamed_file)[
-                                len(self.app.srcdir) + 1 :
-                            ]
-
-                            from_fancy_path = (
-                                '"' + from_fancy_path.replace("\\", "/") + '"'
-                            )
-
-                            to_fancy_path = str(hint_to)[len(self.app.srcdir) + 1 :]
-
-                            to_fancy_path = '"' + to_fancy_path.replace("\\", "/") + '"'
-
-                            file.write(from_fancy_path + " " + to_fancy_path + "\n")
-                            logger.info(
-                                f"{green('(okay)')} Renamed file {renamed_file} has been redirected to {hint_to} in your redirects file!"
-                            )
-                        else:
-                            err_msg = (
-                                f"{red('(broken)')} {renamed_file} was deleted but is not redirected!"
-                                f" Hint: This file was renamed to {hint_to} with a similarity of {perc}%."
-                            )
-                            logger.error(err_msg)
-                            self.app.statuscode = 1
-                else:
-                    err_msg = (
-                        f"{red('(broken)')} {renamed_file} was deleted but is not redirected!"
-                        f" Hint: This file was renamed to {hint_to} with a similarity of {perc}%."
+            for renamed_file in rename_hints:
+                hint_to, perc = rename_hints[renamed_file]
+
+                if renamed_file in absolute_redirects:
+                    logger.info(
+                        f"renamed file {renamed_file} redirects to {absolute_redirects[renamed_file]}."
                     )
-                    logger.error(err_msg)
-                    self.app.statuscode = 1
+                    continue
+
+                if self.name == "rediraffewritediff":
+                    if perc >= self.app.config.rediraffe_auto_redirect_perc:
+                        rel_rename_from = f'"{relpath(renamed_file, src_path)}"'
+                        rel_rename_to = f'"{relpath(hint_to, src_path)}"'
+                        redirects_file.write(f"{rel_rename_from} {rel_rename_to}\n")
+                        logger.info(
+                            f"{green('(okay)')} Renamed file {rel_rename_from} has been redirected to {rel_rename_to} in your redirects file!"
+                        )
```

Comment:
Does this still use forward slashes on windows machines? I think we should standardize on forward slashes.

Reply:
I believe it uses backslashes?
Standardizing should be as simple as changing `relpath(arg0, src_path)` to
```python
str(PurePath(arg0.relative_to(src_path)))
```

I'm not on Windows right now but can try it out tomorrow.